code/datums/mood.dm

```diff
@@ -18,8 +18,11 @@
 	var/sanity = SANITY_NEUTRAL
 	/// the total combined value of all visible moodlets for the mob
 	var/shown_mood
-	/// Moodlet value modifier
-	var/mood_modifier = 1
+	/// Moodlet value modifiers
+	var/mood_modifier = list(
+		MOOD_MODIFIER_GOOD = 1,
+		MOOD_MODIFIER_BAD = 1,
+	)
```

Comment:
This probably doesn't need to be a list. just a `var/good_mood_modifier` and a `var/bad_mood_modifier`. It's not like there's any other moodlet categories than "good" and "bad".

Reply:
i think that's hardcoding for the sake of hardcoding, what if someone has another category of moodlets they wanna fuck w/ down the line?